tests/unittest/test_crashstorage.py

```diff
@@ -38,9 +38,11 @@ def test_flow(self, client):
         # there's only one crash in it and then verify the contents of the crash.
 
         # Verify things got saved
-        assert crashmover.crashstorage.saved_things == [
-            {"crash_id": "de1bb258-cbbf-4589-a673-34f800160918"}
+        crash_ids = [
+            crash_report.crash_id
+            for crash_report in crashmover.crashstorage.saved_things
```

Comment:
Should there also be a test for the noop `load_crash` method?

Reply:
We use this in other parts of the tests so it's implicitly tested. I can add an explicit test.